inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -235,7 +235,132 @@ ie::CNNNetwork FrontEnd::convertNetwork(ie::CNNNetwork& network) {
     IE_SUPPRESS_DEPRECATED_END
 }
 
-std::set<std::string> FrontEnd::checkSupportedLayers(const ie::CNNNetwork& network) {
+std::vector<ie::CNNNetwork> FrontEnd::checkSupportedNetworks(const ie::CNNNetwork& network, std::set<std::string>& namesToExclude) {
+    std::vector<ie::CNNNetwork> supportedNetworks;
+    if (network.getFunction() && network.getFunction()->is_dynamic()) {
+        auto copyNetwork = InferenceEngine::details::cloneNetwork(network);
+        try {
+            convertNetwork(copyNetwork);
+        } catch(vpu::details::VPUException e) {
+            std::string name = e.what();
+            std::string nameDelimiter("node ");
+            name.erase(0, name.find(nameDelimiter) + nameDelimiter.length());
+            nameDelimiter = " of";
+            name.erase(name.find(nameDelimiter), name.length());
```

Comment:
Is there no other way to get the node's name other than getting it from the exception message?

Reply:
Nevermind, I got it.